pulumi/python/kubernetes/applications/sirius/__main__.py

```diff
@@ -64,41 +69,69 @@ def add_namespace(obj):
 
 stack_name = pulumi.get_stack()
 project_name = pulumi.get_project()
-eks_project_name = pulumi_eks_project_name()
+k8_project_name = pulumi_k8_project_name()
 pulumi_user = pulumi_config.get_pulumi_user()
 
-eks_stack_ref_id = f"{pulumi_user}/{eks_project_name}/{stack_name}"
-eks_stack_ref = pulumi.StackReference(eks_stack_ref_id)
-kubeconfig = eks_stack_ref.get_output('kubeconfig').apply(lambda c: str(c))
-eks_stack_ref.get_output('cluster_name').apply(
+k8_stack_ref_id = f"{pulumi_user}/{k8_project_name}/{stack_name}"
+k8_stack_ref = pulumi.StackReference(k8_stack_ref_id)
+kubeconfig = k8_stack_ref.get_output('kubeconfig').apply(lambda c: str(c))
+k8_stack_ref.get_output('cluster_name').apply(
     lambda s: pulumi.log.info(f'Cluster name: {s}'))
 
-k8s_provider = k8s.Provider(resource_name=f'ingress-setup-sample', kubeconfig=kubeconfig)
+k8s_provider = k8s.Provider(resource_name=f'ingress-controller', kubeconfig=kubeconfig)
+
+# TODO: Streamline the logic for FQDN/IP into something a bit more sane and scalable
```

Comment:
Move TODOs to GH issues.

Reply:
Created issues, tagged TODO's with the issue # for now.